client/src/views/Tests/EditTest.tsx

```diff
@@ -0,0 +1,40 @@
+import { models } from '@yahalom-tests/common';
+import React, { useState } from 'react'
+import { SectionNavigator, Section } from '../../components'
+import { TestDetails, TestEmails, TestQuestions } from "./TestForm";
+
+const EditTest: React.FC = () => {
+    const [test, setTest] = useState<models.dtos.TestDto>({
+        title: "",
+        intro: "",
+        questions: [],
+        language: models.enums.Language.Hebrew,
+        teacherEmail: "",
+        minPassGrade: 55,
+        isReviewEnabled: true,
+        successMessage: "",
+        failureMessage: "",
+        successEmail: { body: "", subject: "" },
+        failureEmail: { body: "", subject: "" },
+    });
+
+    /* prop to all - test */
+    /* Test form */
+    return (
+        <form>
+            <SectionNavigator>
+                <Section label="Test Details">
+                    <TestDetails {...test} />
+                </Section>
+                <Section label="Test Complition">
+                    <TestEmails {...test} />
+                </Section>
+                <Section label="Test Questions">
+                    <TestQuestions {...test} />
```

Comment:
Don't really like this `{...test}` thing.
Please check [this](https://github.com/ChemiAtlow/YahalomTests/commit/d6d412ac4884445e16936771503f83fffcaec28c#diff-bb88cd71abdbc573335f41f7e13942cd1f2166b82fe2a1e33542b0f5f7a50408R6) as an example of a better solution.